### DISCUSSION TITLE
佬,格式化直接用esLint , 去掉 prettier

### DISCUSSION THREAD

**Discussion by weiShaoY** [d3george/slash-admin#99] [Ideas]
可以参考下 
https://github.com/antfu-collective/vitesse
https://github.com/antfu/eslint-config

另外, 建议添加 [unplugin-auto-import](https://github.com/unplugin/unplugin-auto-import)  省去引入

**Comment by dingyi214** [comment]
1. 有计划后面会用 biome 替换 eslint + prettier 
2. unplugin-auto-import没用过， 大概率也不会用，因为我觉得目前项目里引入也是自动引入的，而且配合lint 能给引入排序，没有什么维护成本，如果引入了，对于新人来说 很像黑魔法，

**Comment by dfang** [comment]
biome +1, 快，编辑器不卡

**Reply by dingyi214** [reply]
最主要的是 用了 eslint + prettier 后 根目录一堆配置文件， 看着不爽

**Comment by dingyi214** [comment]
已替换
1. replaced eslint + prettier with biome
2. replaced husky + lint-stage with lefthook

**Reply by weiShaoY** [reply]
老哥,麻烦更新下文档

**Comment by dfang** [comment]
nice，lefthook + biome挺好
